lib/xdrgen/ast/declarations/string.rb

```diff
@@ -3,9 +3,11 @@ class String < Base
     delegate :name, to: :identifier
 
     def size
-      size_str = size_spec.size_t.text_value
+      size_spec.size
```

Comment:
As far as I can tell, you are changing semantics here.  XDR strings are not required to have an upper bound (they can be declared like `string<>`)... Am I correct that you've changed that behavior?